drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -315,8 +320,9 @@
 
 .xblock--drag-and-drop--editor .remove-item .icon.remove {
     background: #fff;
+    color: #0072a7;
 }
 .xblock--drag-and-drop--editor .remove-item .icon.remove:before,
 .xblock--drag-and-drop--editor .remove-item .icon.remove:after {
-    background: #2e83cd;
+    background: #236299;
```

Comment:
What is the reason for all these subtle color tweaks? Also some of the new colors look the same to me but are using different hex values - should they not be made more consistent?

![screen shot 2015-12-18 at 11 56 26 am](https://cloud.githubusercontent.com/assets/945577/11906002/774a3d14-a57e-11e5-8697-60578f7bd9c1.png)